Tests/ArgumentParserEndToEndTests/UnparsedValuesEndToEndTest.swift

```diff
@@ -57,6 +57,73 @@ extension UnparsedValuesEndToEndTests {
   }
 }
 
+// MARK: Two value + unparsed optional variable
+
+fileprivate struct Hogeraa: ParsableArguments {
+  var fullName: String? = "Full Name"
+}
+
+fileprivate struct Hogera: ParsableArguments {
```

Comment:
If we move structs into the extension code block then we could reuse names for different test collections. This could cut down on the amount of metasyntactic type names (and modifications of those names) used.

Reply:
For sure! I don't think tests are very consistent around this, so if you'd like to structure yours that way, that'd be great.